proto/babylon/finality/v1/params.proto

```diff
@@ -30,4 +30,7 @@ message Params {
   // jail_duration is the minimum period of time that a finality provider remains jailed
   google.protobuf.Duration jail_duration = 5
   [(gogoproto.nullable) = false, (amino.dont_omitempty) = true, (gogoproto.stdduration) = true];
+  // activation_block_height is the babylon block which the finality module will start to
+  // accept public randomness commit and finality voting.
+  uint64 activation_block_height = 6;
```

Comment:
```suggestion
  uint64 finality_activation_height = 6;
```

This is because the BTC staking activation height will be moved to the Finality module ([ref](https://github.com/babylonlabs-io/babylon/pull/216)), the BTC staking module will fall back to a storage for BTC delegations and FPs (for Babylon and all consumer chains). Better to distinguish between BTC staking activation v.s. finality activation. Same comments for other "activation" usage